package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "react-native-onesignal",
-  "version": "3.9.1",
+  "version": "4.0.0-beta.0",
```

Comment:
Is `-beta.0` the convention for npm?
Also is starting at `0` a convention for npm?